concordium-contracts-common/src/schema_json.rs

```diff
@@ -964,6 +999,109 @@ impl Type {
+                outer_map.insert("Enum".to_string(), serde_json::Value::Array(vector));
+
+                serde_json::Value::Object(outer_map)
+            }
+            Self::TaggedEnum(tagged_enum) => {
+                let mut outer_map = Map::new();
+
+                let mut vector = Vec::new();
+
+                for (_tag, (name, field)) in tagged_enum.iter() {
+                    let mut inner_map = Map::new();
+                    inner_map.insert(name.to_string(), field.field_to_json());
+                    vector.push(serde_json::Value::Object(inner_map))
+                }
+                outer_map.insert("Enum".to_string(), serde_json::Value::Array(vector));
+
+                serde_json::Value::Object(outer_map)
+            }
+            Self::Struct(field) => field.field_to_json(),
+            Self::ByteList(_) => {
+                serde_json::Value::String("<String with lowercase hex>".to_string())
+            }
+            Self::ByteArray(_) => {
+                serde_json::Value::String("<String with lowercase hex>".to_string())
+            }
+            Self::String(_) => serde_json::Value::String("<String>".to_string()),
+            Self::Unit => serde_json::Value::Array(Vec::new()),
+            Self::Bool => serde_json::Value::String("<Bool>".to_string()),
+            Self::U8 => serde_json::Value::String("<UInt8>".to_string()),
+            Self::U16 => serde_json::Value::String("<UInt16>".to_string()),
+            Self::U32 => serde_json::Value::String("<UInt32>".to_string()),
+            Self::U64 => serde_json::Value::String("<UInt64>".to_string()),
+            Self::U128 => serde_json::Value::String("<UInt128>".to_string()),
+            Self::I8 => serde_json::Value::String("<Int8>".to_string()),
+            Self::I16 => serde_json::Value::String("<Int16>".to_string()),
+            Self::I32 => serde_json::Value::String("<Int32>".to_string()),
+            Self::I64 => serde_json::Value::String("<Int64>".to_string()),
+            Self::I128 => serde_json::Value::String("<Int128>".to_string()),
+            Self::ILeb128(_) => {
+                serde_json::Value::String("<String with signed integer>".to_string())
+            }
+            Self::ULeb128(_) => {
+                serde_json::Value::String("<String with unsigned integer>".to_string())
+            }
+            Self::Amount => serde_json::Value::String("<Amount>".to_string()),
+            Self::AccountAddress => serde_json::Value::String("<AccountAddress>".to_string()),
+            Self::ContractAddress => {
+                let mut contract_address = Map::new();
+                contract_address.insert("index".to_string(), Type::U64.schema_type_to_json());
+                contract_address.insert("subindex".to_string(), Type::U64.schema_type_to_json());
+                serde_json::Value::Object(contract_address)
+            }
+            Self::Timestamp => serde_json::Value::String("<Timestamp>".to_string()),
```

Comment:
An example for the date format would be nice for the user. Like `"2023-12-01T12:00:00Z"`.

Reply:
Yes, that would be nice. It is a bit different than what the client does, but it could be updated as well.
I still think the brackets make sense to indicate that it is a placeholder. Perhaps `"<Timestamp ("2000-12-24T12:00:00Z")>"`?

Reply:
It is displayed as follows now:
```
<Timestamp (e.g. `2000-01-01T12:00:00Z`)>
```